core/Controller/PreviewController.php

```diff
@@ -107,7 +121,15 @@ public function getPreviewByFileId(
 	}
 
 	/**
-	 * @return DataResponse|FileDisplayResponse
+	 * @param Node $node
+	 * @param int $x
+	 * @param int $y
+	 * @param bool $a
+	 * @param bool $forceIcon
+	 * @param string $mode
+	 * @return Response
+	 * @throws NotFoundException
+	 * @throws InvalidPathException
```

Comment:
```suggestion
	 * @return DataResponse|FileDisplayResponse
```

or you remove the whole doc block and change the end of the signature to:
```
string $mode) : DataResponse|FileDisplayResponse {
```

Reply:
See this